src/connection/manager.ts

```diff
@@ -112,15 +113,19 @@ export class SQLJobManager {
       return content.runSQL(queryContext) as Promise<T[]>;
     }
   }
-  getPagingStatement<T>(query: string, opts?: QueryOptions): Query<T> {
+  async getPagingStatement<T>(query: string, opts?: QueryOptions): Promise<Query<T>> {
     const selected = this.jobs[this.selectedJob]
     if (ServerComponent.isInstalled() && selected) {
       return selected.job.query<T>(query, opts);
     } else if(!ServerComponent.isInstalled()) {
-      ServerComponent.checkForUpdate();
+      let updateResult = await ServerComponent.checkForUpdate();
+      if(UpdateStatus.JUST_UPDATED === updateResult) {
+        setTimeout(() => {JobManagerView.setVisible(true)});
+        return this.getPagingStatement(query, opts);
+      }
       throw new Error(`Database server component is required. Please see documentation for details.`);
     }else {
-      throw new Error(`Active SQL job is required. Please spin one up first.`);
+      throw new Error(`Active SQL job is required. Please spin one up in the 'SQL Job Manager' view and try again.`);
```

Comment:
Wondering why we couldn't spin up the job for them.  If they have multiple JDBC configs saved we wouldn't know which to choose, but it they had one or none, seems like we could maybe start the job.  Just a thought, no action required.

Code looks good, tested the flow and it seems fine.

Reply:
I agree, but we should only do so if the autostart setting is set. change at https://github.com/halcyon-tech/vscode-db2i/pull/74/commits/dae5229fd8e159bfba19b83597e6ea72f2d6cd12